### PR TITLE
[CI] Fix prettier

### PR DESCRIPTION
**Pull request by SaschaMann**
It's a mess and works on master but not PRs and everything's on a different version.

This hopefully fixes that.

### CODE REVIEW

File: .github/workflows/format-code.yml [hunk 1]
```diff
@@ -48,12 +48,9 @@ jobs:
...
       - name: 'Format code'
-        run: |
-          # Use prettier to format the added or modified Markdown files in the PR (if any)
-          jq -r '.issue.pull_request.url | . += "/files"' "$GITHUB_EVENT_PATH" | \
-            xargs curl --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' --url | \
-            jq -c '.[] | select(.status == "added" or .status == "modified") | select(.filename | match("\\.(md|json)$")) | .filename' | \
-            xargs npx prettier@2.0.4 --write
+        run: ./bin/format.sh
+        env:
+          EXERCISM_PRETTIER_VERSION: '2.1.2'
```
Comment (SaschaMann) [inline .github/workflows/format-code.yml:53]: I wonder if we should set this as a secret so that it can't possibly get out of sync ever.

**Comment by SaschaMann** [comment]
The slash command doesn'tw ork in here because it runs on the workflow on the master branch. I've verified on a fork that it works, though.

File: languages/julia/concepts/iterator-protocol/about.md [hunk 2]
```diff
@@ -7,12 +7,12 @@ The first method will return the first item and state, while the second method w
...
-    Alternatively, you can work your way through `MethodErrors` to find out which methods you need to implement:
-    <!-- TODO check if code blocks work within info boxes -->
-    ```julia
-    julia> struct MyIter end;
+Julia currently doesn't have a way to formally define interfaces.
+To implement an interface, you need to look up which methods need to be defined in the Manual, or in the documentation of the package that "defines" the interface.
+Alternatively, you can work your way through `MethodErrors` to find out which methods you need to implement:
+<!-- TODO check if code blocks work within info boxes -->
+```julia
+julia> struct MyIter end;
```
Comment (SaschaMann) [inline languages/julia/concepts/iterator-protocol/about.md:15]: This is wrong.

File: languages/julia/concepts/iterator-protocol/about.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ The iteration ends if `iterate` returns `nothing`.
     ```
 
 !!! note
-    The iterator object itself is usually not mutated by iteration.
+The iterator object itself is usually not mutated by iteration.
```
Comment (SaschaMann) [inline languages/julia/concepts/iterator-protocol/about.md:25]: So is this.

File: languages/julia/exercises/concept/fibonacci-iterator/.docs/instructions.md [hunk 4]
```diff
@@ -15,13 +15,14 @@ $$
...
+
 $$
 
 For example, the third element would be $a_3 = a_1 + a_2 = 1 + 1 = 2$, the fourth would be $a_4 = a_2 + a_3 = 1 + 2 = 3$, and so on.
 
 !!! note
-    Some sources define the first elements as $a_0 = 0$ and $a_1 = 1$.
-    However in the context of this exercise, we define the sequence without a $0$-th element.
+Some sources define the first elements as $a_0 = 0$ and $a_1 = 1$.
+However in the context of this exercise, we define the sequence without a $0$-th element.
```
Comment (SaschaMann) [inline languages/julia/exercises/concept/fibonacci-iterator/.docs/instructions.md:25]: and this.

**Review by SleeplessByte — APPROVED** [review]

**Review by ErikSchierboom — COMMENTED** [review]
Thanks for doing this @SaschaMann! We should update https://github.com/exercism/v3/blob/master/docs/maintainers/style-guide.md#3-using-a-script too, as `format.cmd` is no longer available and `format.sh` requires the env variable. We might want to just entirely remove the section or else print a warning in `format.sh` when the env var is not set.

**Pull request merged**